matter/efr32/efr32mg24/BRD2601B/autogen/sl_event_handler.c

```diff
@@ -1,38 +1,43 @@
 #include "sl_event_handler.h"
 
-#include "cmsis_os2.h"
 #include "em_chip.h"
-#include "gpiointerrupt.h"
-#include "pa_conversions_efr32.h"
-#include "sl_board_control.h"
+#include "sl_device_init_nvic.h"
 #include "sl_board_init.h"
-#include "sl_device_init_clocks.h"
 #include "sl_device_init_dcdc.h"
-#include "sl_device_init_emu.h"
+#include "sl_hfxo_manager.h"
 #include "sl_device_init_hfxo.h"
+#include "sl_device_init_lfrco.h"
 #include "sl_device_init_lfxo.h"
-#include "sl_device_init_nvic.h"
-#include "sl_hfxo_manager.h"
+#include "sl_device_init_clocks.h"
+#include "sl_device_init_emu.h"
+#include "pa_conversions_efr32.h"
 #include "sl_rail_util_pti.h"
+#include "sl_board_control.h"
 #include "sl_bt_rtos_adaptation.h"
+#include "sl_sleeptimer.h"
+#include "gpiointerrupt.h"
 #include "sl_simple_button_instances.h"
 #include "sl_simple_led_instances.h"
-#include "sl_sleeptimer.h"
 #if defined(CONFIG_ENABLE_UART)
 #include "sl_uartdrv_instances.h"
 #endif // CONFIG_ENABLE_UART
+#include "psa/crypto.h"
+#include "sli_protocol_crypto.h"
+#include "cmsis_os2.h"
 #include "sl_bluetooth.h"
 #if defined(SL_CATALOG_POWER_MANAGER_PRESENT)
 #include "sl_power_manager.h"
 #endif
 
-void sl_platform_init(void) {
+void sl_platform_init(void)
+{
   CHIP_Init();
   sl_device_init_nvic();
   sl_board_preinit();
   sl_device_init_dcdc();
   sl_hfxo_manager_init_hardware();
   sl_device_init_hfxo();
+  sl_device_init_lfrco();
```

Comment:
Why this one ads lfrco and why it has both lfrco and lfxo

Reply:
slc says its necessary